Formula/w/wush.rb

```diff
@@ -0,0 +1,17 @@
+class Wush < Formula
+  desc "Simplest & fastest way to transfer files between computers via wireguard"
+  homepage "https://github.com/coder/wush"
+  url "https://github.com/coder/wush/archive/refs/tags/v0.1.2.tar.gz"
+  sha256 "8e1ffd24b51775ac68116b381cf4523a54d4e7742e440b7d279418faa52904eb"
+  license "CC0-1.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}"), "./cmd/..."
+  end
+
+  test do
+    system bin/"wush", "version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I'll try to implement that, but commands with wush run until killed, so I referred sshfs for the test case. I need an example formula which implements more test cases while running never ending commands.

Reply:
How about httpd? That runs a (never ending) server

Reply:
Yes, I did something similar, thanks for suggesting. I am in process of enhancing that a little more.